MiniApp/Classes/core/Extensions/Int+MiniApp.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+
+internal extension Int {
+    func compare(to anotherInt: Self) -> Int {
+        if self != anotherInt {
+            if self > anotherInt {
+                return 1
+            } else {
+                return -1
+            }
+        }
+        return 0
+    }
+}
```

Comment:
the method mimics Java comparable methods. If the compared `Int` is _greater_ it will return `1`, _lesser_ will return `-1` and _equal_ will return `0`